### ISSUE TITLE
reMarkable 2 stuck at is starting after factory reset after update, can ssh

### ISSUE THREAD

**Issue by antonAtananasov** [ddvk/remarkable2-framebuffer#74]
Hi all,

Yesterday, the reMarkable displayed an update available on the bottom of the screen, asking for restart. I had the automatic installision of this nice thingy [here](https://github.com/ddvk/remarkable-hacks#automagic). 

When the update restarted the device, the addional features were gone and i could not install them again because they no longer support reMarkable's new version, so I decided to make a factory reset, hoping it to downgrade a few versions (since i got my tab just 2 days ago) and that I could get the features back. 

Now, it's stuck on "is starting" but I can see it refresh the screen if i shut it off (hold button for 15 secs) and then on (hold for 2 or 3 sec). I can still ssh it with a pc, trying to run xochitl gives me "Unable to write device id".

**Tl;dr** tried to install remarkable-hacks, 
got update that disabled them and did factory reset, 
stuck on "is starting", still able to ssh.

[xochitl output.txt](https://github.com/ddvk/remarkable2-framebuffer/files/7261312/xochitl.output.txt)

Many thanks!

**Comment by Eeems** [comment]
I would recommend against doing a factory reset in the future. There is no need to if you want to downgrade to an older version (https://github.com/ddvk/remarkable-update).

I'd recommend checking to see if your other partition has a working install in it and switching to it if it does and then upgrading to the latest again.
```bash
mount /dev/mmcblk1p$(fw_printenv -n fallback_partition) /mnt
ls /mnt
```
This should let you know if you have content in the other partition. If you do, you can use https://github.com/ddvk/remarkable-update/blob/main/switch.sh to switch partitions, and then reboot.

Otherwise you will need to get an update image and copy over the files missing from your current install, or just overwrite the fallback partition with it and switch.

**Comment by antonAtananasov** [comment]
The result of the command is that there is no such device
`mount: /mnt: special device /dev/mmcblk1p2 does not exist.`

Any other solutions?

**Comment by Eeems** [comment]
Oh interesting, that would mean that the partition doesn't exist? What does `ls /dev/mmcblk*` return?

**Comment by antonAtananasov** [comment]
This is what I get:
`root@reMarkable:~# ls /dev/mmcblk*
/dev/mmcblk2       /dev/mmcblk2boot1  /dev/mmcblk2p2     /dev/mmcblk2p4
/dev/mmcblk2boot0  /dev/mmcblk2p1     /dev/mmcblk2p3     /dev/mmcblk2rpmb
`

**Comment by antonAtananasov** [comment]
Maybe the command should be 
`mount /dev/mmcblk**2**p$(fw_printenv -n fallback_partition) /mnt
ls /mnt`
because it's a rM2?

**Comment by antonAtananasov** [comment]
The operation with switching the partitions was successful in the end and i am happy ヾ（＾ヮ＾)ﾉ
Thank you very much for the help! You practically fixed it in less than 5 minutes!

**Comment by Eeems** [comment]
> 
> 
> > mount /dev/mmcblk1p$(fw_printenv -n fallback_partition) /mnt
> > ls /mnt
> 
> Maybe the command should be `mount /dev/mmcblk2p$(fw_printenv -n fallback_partition) /mnt && ls /mnt` because it's a rM2? It runs successfully and returns this: `bin lib proc uboot-postinst boot lost+found run uboot-version dev media sbin usr etc mnt sys var home postinst tmp`

I guess they have it mapped differently on the rM2 than the rM1

> 
> 
> The operation with switching the partitions was successful in the end and i am happy ヾ（＾ヮ＾)ﾉ Thank you very much for the help! You practically fixed it in less than 5 minutes!

Awesome! Glad you are back up and running!
